javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -1281,6 +1291,7 @@ private predicate summarizedHigherOrderCall(
     DataFlow::Node innerArg, DataFlow::SourceNode cbParm, PathSummary oldSummary
   |
     reachableFromInput(f, outer, arg, innerArg, cfg, oldSummary) and
+    not arg = DataFlow::capturedVariableNode(_) and // Only track actual parameter flow
```

Comment:
I would like a test motivating this change.

Reply:
Expanded on the comment and added a test to demonstrate that we don't lose any flow.